### ISSUE TITLE
Transparency material does not work well with material used for display of lines wires

### ISSUE THREAD

**Issue by uzername** [anycad/anycad.rapid.net.sample#21]
I have placed a scenenode with opacity using such code:

```
            materialBox = ShapeBuilder.MakeBox(GP.XOY(), Math.Abs( x ), Math.Abs( y ), Math.Abs( z ));
            MeshPhongMaterial GrayMaterial222 = MeshPhongMaterial.Create("phong.color");
            GrayMaterial222.SetColor(Vector3.LightGray);
            GrayMaterial222.SetTransparent(true);
            GrayMaterial222.SetOpacity(0.5f);

            materialNode = BrepSceneNode.Create(materialBox, GrayMaterial222, null);
            mRenderView.ShowSceneNode(materialNode);
```

I have placed some lines above and below it too. It is expected that you can see all those lines, but you cannot.

View of One side:

![image](https://user-images.githubusercontent.com/864126/96544301-f8a36780-12ae-11eb-8fd5-29cda83015d2.png)

View of another side:

![image](https://user-images.githubusercontent.com/864126/96544351-17a1f980-12af-11eb-9f60-a72e40452169.png)

**Comment by anycad** [comment]
Fixed in 20.10.27, please verify it.

**Comment by uzername** [comment]
yes, it is fine now
